packages/js/github-actions/actions/get-plugin-releases/src/get-plugin-releases.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import fetch from 'node-fetch';
+import core from '@actions/core';
+
+/**
+ * Internal dependencies
+ */
+import handleActionErrors from '../../../utils/handle-action-errors';
+
+async function getPluginReleases() {
+	const slug = getInput( 'slug' );
+	const apiEndpoint = getAPIEndpoint( slug );
+
+	fetch( apiEndpoint )
+		.then( ( res ) => res.json() )
+		.then( parsePluginVersions );
+}
+
+function getAPIEndpoint( slug ) {
+	if ( slug === 'wordpress' ) {
+		return 'https://api.wordpress.org/core/version-check/1.7/';
+	}
+
+	return `https://api.wordpress.org/plugins/info/1.0/${ slug }.json`;
+}
+
+function getInput( key ) {
+	const rawInput = core.getInput( key );
+	let input = rawInput;
+
+	if ( rawInput === 'false' ) {
+		input = false;
+	} else if ( rawInput === 'true' ) {
+		input = true;
+	}
+
+	return input;
+}
+
+function setOutput( key, value ) {
+	core.info( `==> Output "${ key }":\n${ value }` );
+	core.setOutput( key, value );
+}
+
+function parsePluginVersions( releases = {} ) {
+	const slug = getInput( 'slug' );
+	const numberOfReleases = parseInt( getInput( 'releases' ), 10 ) || 3;
+	const includeRC = getInput( 'includeRC' ) || false;
+	const includePatches = getInput( 'includePatches' ) || false;
+
+	const output = [];
+
+	if ( slug !== 'wordpress' ) {
+		Object.keys( releases.versions )
+			.reverse()
+			.forEach( ( version ) => {
+				if ( output.length === numberOfReleases ) {
+					return;
+				}
+
+				if (
+					version !== 'other' &&
+					version !== 'trunk' &&
+					! version.includes( 'beta' ) &&
+					! includesRC( version, includeRC ) &&
```

Comment:
The fact that for "1.2.3-rc.1" "includesRC" may return `false` is quite confusing IMHO. 
```js
( includeRC || ! includesRC( version ) )
```
would be more straightforward to me.

Also, "**includesRC**( release.version, **includeRC** )" gives me a bit of :thinking:, and I think it is pretty bug-prone (at least to my dyslectic self;) ) to miss just `s`